chrome_gpu_bug_count.sh

```diff
@@ -7,7 +7,7 @@ BASEDIR=$(dirname "$0")
 INPUT="$BASEDIR/chrome_gpu_triage.json"
 SCRIPT="$BASEDIR/issue_counter.py"
 mkdir -p "$OUTDIR"
-OUTFILE="$OUTDIR/$(date --iso-8601).json"
+OUTFILE="$OUTDIR/$(date "+%Y-%m-%d-%H:%M").json"
```

Comment:
Yeah, if you don't mind updating your branch to have that, I'd appreciate it.  I find time to be kind of noisy in filenames.  If you'd prefer keeping time and are also using this script, I'm happy to change too.